distros/debian10/install_phpmyadmin.sh

```diff
@@ -0,0 +1,89 @@
+#---------------------------------------------------------------------
+# Function: InstallphpMyAdmin
+#    Install and configure phpMyAdmin
+#---------------------------------------------------------------------
+InstallphpmyAdmin() {
+    echo -n "Install myPhpAdmin...."
```

Comment:
Installing phpMyAdmin...   

would be better

Reply:
done

Reply:
actually i removed as its in the webserver script that calls the installation of phpmyadmin